ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -216,6 +221,14 @@
        context
        (ingest-events/provider-autocomplete-suggestion-reindexing-event provider)))))
 
+(defn trigger-bulk-granule-task-status-updates
+  [context]
+  (when-let [incomplete-tasks (seq (granule-bulk-update/get-incomplete-granule-task-ids context))]
+    (doseq [task-id incomplete-tasks]
+      (ingest-events/publish-gran-bulk-update-event
```

Comment:
There is no need to update task status via a message queue event. We should just call the function that does the work here.